scripts/seed.bat

```diff
@@ -4,4 +4,4 @@ python manage.py loaddata accounts
 python manage.py loaddata resources
 python manage.py loaddata profiles
 python manage.py loaddata emailaddresses
-python manage.py loaddata hackathons
+python manage.py loaddata hackathons
```

Comment:
Add back in the newline please

Reply:
🙄🙄Not sure how I did that..... 🤷‍♂️. I've re-added it back in now.